service/rpccommon/server.go

```diff
@@ -307,10 +296,8 @@ func (s *ServerImpl) serveJSONCodec(conn io.ReadWriteCloser) {
 		}
 
 		if mtype.Synchronous {
-			if s.log {
-				argvbytes, _ := json.Marshal(argv.Interface())
-				log.Printf("-> %s(%T%s)\n", req.ServiceMethod, argv.Interface(), argvbytes)
-			}
+			argvbytes, _ := json.Marshal(argv.Interface())
```

Comment:
This is a fairly expensive operation, I don't think we should do it unconditionally given that rpc logging will mostly be disabled. Same goes for the other json.Marshal calls in this function and RPCCallback.Return.